markdown/ch2.markdown

```diff
@@ -469,7 +469,7 @@ Note that some data is *right-censored*, which means that in some cases, we don'
 We count these patients as "alive" for the duration of the survival curve, but more sophisticated analyses might try to estimate their likely time of death.
 
 To obtain a survival curve from survival times, we create a step function that decreases by $1/n$ at each step, where $n$ is the number of patients in the group.
-We then match that function against the non-censored survival times.
+We then match that function against the noncensored survival times.
```

Comment:
uncensored?

Reply:
I actually think non-censored is the correct term here. (Uncensored sounds very wrong LOL) Shall I revert it?

Reply:
If so, I think this is correct.  Thanks for clarifying!